images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -22,10 +22,13 @@ $markdown += New-MDList -Style Unordered -Lines @(
     "Image Version: $env:ImageVersion"
 )
 
-$markdown += New-MDHeader "Enabled windows optional features" -Level 2
-$markdown += New-MDList -Style Unordered -Lines @(
-    "Windows Subsystem for Linux"
-)
+if ($OSName -match "Microsoft Windows Server 2019 Datacenter")
```

Comment:
There is a function: ` Test-IsWin19`. I believe we should use it
https://github.com/actions/virtual-environments/blob/master/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L440

Reply:
Thanks! Replaced.